src/BPGates.jl

```diff
@@ -507,6 +522,7 @@ end
 function QuantumClifford.apply!(state::BellState, g::PauliNoiseOp)
     i = g.idx
     # TODO repetition with ...NoisyReset and PauliNoise...
+    # ^ ?
```

Comment:
this todo is a reminder that there is some code repetition with things implemented in QuantumClifford